xbmc/powermanagement/PowerManager.cpp

```diff
@@ -52,9 +52,11 @@
 #include "linux/LogindUPowerSyscall.h"
 #include "linux/UPowerSyscall.h"
 #endif // HAS_DBUS
-#elif defined(TARGET_WINDOWS)
+#elif defined(TARGET_WINDOWS_DESKTOP)
 #include "powermanagement/windows/Win32PowerSyscall.h"
 extern HWND g_hWnd;
+#elif defined(TARGET_WINDOWS_STORE)
+#include "powermanagement/win10/PowerSyscallWin10.h"
```

Comment:
maybe name the file `Win10PowerSyscall.h` to keep it consistent, or rename the Win32 file, both works